packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7274,25 +7275,31 @@ class KeyedSubtree extends StatelessWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=xXNOkIuSYuA}
 ///
-/// This widget is a simple inline alternative to defining a [StatelessWidget]
-/// subclass. For example a widget defined and used like this:
+/// This widget is an inline alternative to defining a [StatelessWidget]
+/// subclass. For example, instead of defining a widget as follows:
 ///
 /// ```dart
 /// class Foo extends StatelessWidget {
+///   const Foo({super.key});
 ///   @override
-///   Widget build(BuildContext context) => Text('foo');
+///   Widget build(BuildContext context) => const Text('foo');
 /// }
+/// ```
 ///
-/// Center(child: Foo())
+/// ...and using it in the usual way:
+///
+/// ```dart
+/// // continuing from previous example...
```

Comment:
We should somewhere document what strings with special meaning you can use so we can point people to it in code reviews. I wonder, what would be a good place for that.

Reply:
I see that they are documented in analyze_snippet_code.dart.